cmd/minishift/cmd/oc_env.go

```diff
@@ -58,10 +61,36 @@ func getOcShellConfig(ocPath, forcedShell string) (*OcShellConfig, error) {
 	cmdLine := "minishift oc-env"
 	shellCfg := &OcShellConfig{
 		OcDirPath: filepath.Dir(ocPath),
-		UsageHint: shell.GenerateUsageHint(userShell, cmdLine),
 	}
 
-	shellCfg.Prefix, shellCfg.Suffix, shellCfg.Delimiter = shell.GetPrefixSuffixDelimiterForSet(userShell, true)
+	if noProxy {
```

Comment:
Will get rid of this duplicated code with a function.

Reply:
Having this as part of method will make sense so this can be used for docker env and oc env both.

Reply:
Done.